packages/grid/x-data-grid-pro/src/tests/rowGrouping.DataGridPro.test.tsx

```diff
@@ -104,7 +104,7 @@ describe('<DataGridPro /> - Group Rows By Column', () => {
       });
     });
 
-    describe('prop: rowGroupingModel', () => {
+    describe('props: rowGroupingModel', () => {
```

Comment:
The core repo uses `prop: propName`: https://github.com/mui/material-ui/blob/32fd4a3249fb1de139b17ed54bd54802b3d50f44/packages/mui-material/src/Autocomplete/Autocomplete.test.js#L112

Reply:
I can put `prop: propName` to be consistent with the core then

Reply:
Done